modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -111,6 +113,15 @@ def validate_line_of_business(lob, submission_version)
       end
     end
 
+    def validate_zip_code(zip_code)
+      unless VALID_ZIP_CODE_REGEX.match?(zip_code)
+        # rubocop:disable Layout/LineLength
+        detail = 'Zip code must be a string of either five digits ("XXXXX") or five digits followed by a hyphen and four more digits ("XXXXX-XXXX"). Use "00000" for Veterans with non-US addresses.'
+        # rubocop:enable Layout/LineLength
```

Comment:
Not at all necessary before merging, but if you want to avoid having to disable Rubocop for this line, you could do the following using `\` (I've used this approach before and Rubocop accepts it):

```
detail = 'Zip code must be a string of either five digits ("XXXXX") or five digits followed by a hyphen ' \
         'and four more digits ("XXXXX-XXXX"). Use "00000" for addresses outside the US.'
```